pset-functions.js

```diff
@@ -31,6 +31,20 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+  if (playerCardScore > 21 && dealerCardScore > 21) {
+    return 0
+  } else if (playerCardScore > 21) {
+    return dealerCardScore
+  } else if (dealerCardScore > 21) {
+    return playerCardScore
+  } else {
+    return Math.max(playerCardScore, dealerCardScore)
```

Comment:
You if/else cascade is flowing perfectly! Great use of Math methods for the final catch all.